keyboards/xd75/keymaps/scheiklp/rules.mk

```
@@ -0,0 +1,25 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+DYNAMIC_MACRO_ENABLE = yes  # Enable dynamically recorded macros
+AUTO_SHIFT_ENABLE    = yes  # Tap a key for lowercase and hold it for upper case
+KEY_LOCK_ENABLE      = yes  # Adds a lock key to continually hold down a key
+TAP_DANCE_ENABLE     = no   # Enable TapDance functionality
+BOOTMAGIC_ENABLE     = no    # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE      = yes    # Mouse keys(+4700)
+EXTRAKEY_ENABLE     = yes    # Audio control and System control(+450)
+CONSOLE_ENABLE      = no    # Console for debug(+400)
+COMMAND_ENABLE      = no    # Commands for debug and configuration
+NKRO_ENABLE         = yes   # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+USB_6KRO_ENABLE     = no    # 6key Rollover
+BACKLIGHT_ENABLE    = no    # Enable keyboard backlight functionality
+MIDI_ENABLE         = no    # MIDI controls
+AUDIO_ENABLE        = no    # Audio output on port C6
+#VARIABLE_TRACE      = no    # Debug changes to variable values
+UNICODE_ENABLE      = no    # Unicode (can't be used with unicodemap)
+UNICODEMAP_ENABLE   = no   # Enable extended unicode
+BLUETOOTH_ENABLE    = no    # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE     = no    # Enable WS2812 RGB underlight. 
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE    = no    # Breathing sleep LED during USB suspend
```

This can be condensed down to:
```suggestion
BOOTMAGIC_ENABLE = no
COMMAND_ENABLE = no
RGBLIGHT_ENABLE = no
DYNAMIC_MACRO_ENABLE = yes
AUTO_SHIFT_ENABLE = yes
KEY_LOCK_ENABLE = yes
```